indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskClient.java

```diff
@@ -224,6 +231,31 @@ public DateTime getStartTime(final String id)
     }
   }
 
+  public List<ParseExceptionReport> getParseErrors(final String id)
+  {
+    log.debug("getParseErrors task[%s]", id);
+
+    try {
+      final StringFullResponseHolder response = submitRequestWithEmptyContent(
+          id,
+          HttpMethod.GET,
+          "unparseableEvents",
+          null,
+          true
+      );
+      return response.getContent() == null || response.getContent().isEmpty()
+             ? Collections.emptyList()
+             : deserialize(response.getContent(), TYPE_REFERENCE_LIST_PARSE_EXCEPTION_REPORT);
+    }
+    catch (NoTaskLocationException e) {
+      return Collections.emptyList();
```

Comment:
will this be logged anywhere as a warning?  I have same question for situation when respose content is null or empty.

Reply:
I adjusted this and the above moving average method to log warnings